### DISCUSSION TITLE
Dreame Account login not possible?

### DISCUSSION THREAD

**Discussion by WildEchoWanderer** [Tasshack/dreame-vacuum#568] [General]
I had this Integration For a few month with a dreame L20 ultra working perfectly.
Yesterday i checked hacs and realised the repo is no longer visible. I then deleted it and downloaded the New found repo

When i now try to newly Setup the deleted device i get a prompt to login Into xiaomi Account and not dreame. 

Why? Is this intended?
How to get my Dreame L20 ultra working again?

Edit: i guess it was the Beta channel

**Comment by WildEchoWanderer** [comment]
Also can't Download beta Branch anymore

https://github.com/Tasshack/dreame-vacuum/assets/116340711/b5b4ebbc-506f-4831-b7f2-1cae7d069bb8

**Comment by Tasshack** [comment]
This is an HACS issue and I think you should create an issue about this on its repository so developers can inform me if there is something can do about it or it just needs to be fixed from HACS.

**Reply by WildEchoWanderer** [reply]
Thanks i will do this
